test/parallel/test-freeze-intrinsics.js

```diff
@@ -3,8 +3,7 @@
 require('../common');
 const assert = require('assert');
 
-try {
-  Object.defineProperty = 'asdf';
-  assert(false);
-} catch {
-}
+assert.throws(
+  () => Object.defineProperty = 'asdf',
```

Comment:
Ignorable style micro-nit that others might not agree with anyway: I prefer braces (`{` and `}`) in situations like this because without braces, the arrow function returns a value, and IMO we shouldn't return a value unless we are intending to return a value. Here, we are returning a value as a style side effect. Someone reading this might not know if the return value is significant. By adding braces, we make it clear it's not.

There are, of course arguments the other way. (Someone who knows how `assert.throws()` works will know the return value is irrelevant. Return a value or not, who cares really?) Just my opinion. I think...uh...@sam-github maybe? Someone else thinks the same way. But it's hardly universal on the project....

Anyway, code change looks good with or without this suggestion. But if I've managed to convince you, cool. 😆

Reply:
I agree with you, but since the assignment throws an error, there is in fact no return value 😁